templates/admin/cloner-suggestions.html

```diff
@@ -0,0 +1,34 @@
+<div class="book-card">
```

Comment:
More about this in https://www.algolia.com/doc/guides/building-search-ui/widgets/customize-an-existing-widget/js/#templating-your-ui